pkg/storaged/side-panel.jsx

```diff
@@ -104,7 +107,7 @@ export class SidePanelRow extends React.Component {
                     {this.props.actions}
                 </div>);
         else if (client.path_jobs[job_path])
-            decoration = <div className="sidepanel-row-decoration spinner spinner-sm" />;
+            decoration = <Spinner size="sm" />;
         else if (client.path_warnings[job_path])
             decoration = <div className="sidepanel-row-decoration pficon pficon-warning-triangle-o" />;
```

Comment:
It is this line that I get there rendered.
Still looks the same though :/

Reply:
Are you sure :) ?

I recompiled it and it looks like this:

![Screen Shot 2020-10-13 at 16 28 15](https://user-images.githubusercontent.com/14921356/95874409-277e8280-0d71-11eb-84b3-45cc79220b32.png)

Reply:
Hmm, maybe I failed to rebuild it? I would try again, but there are not the correct changes